tools/monitoring/README.md

```diff
@@ -0,0 +1,22 @@
+# Knative Monitoring
+
+Knative monitoring tool checks the prow job status and scrape through all the failure logs to catch test infrastructure
+failures.
+
+## Setup
+
+### Create the cluster
+
+```bash
+gcloud container clusters create monitoring --enable-ip-alias
+```
+
+### Secrets
+
+* `monitoring-db-credentials`: monitoring database user name and password
```

Comment:
Where is this var used?

Reply:
is this the mounted secret?

Reply:
Sorry missed this comment. Yes this is the secret name that you'll see when you run `kubectl get secrets`

Reply:
What is the purpose of putting the "secrets" section here? Is it to explain what does the field mean in the yaml, or is it to remind the user to set up the secret? Please be more explicit about it

Reply:
It is to remind the user to set up the secrets. Removed this section per offline conversation.